indico/web/client/js/react/components/WTFSearchDropdown.module.scss

```diff
@@ -0,0 +1,15 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+:global(.ui.multiple.search.dropdown) :global(input[type='text'].search) {
```

Comment:
You are changing global styling for ALL dropdowns.

Add a normal class here and include that with `styleName="..."` in your jsx file. That way you keep it limited to this component.

Reply:
oh, I thought since it was defined in this module it will only affect this component. I've changed it now

Reply:
Yes, that's the case for normal classes. Adding `:global(...)` around a class disables the CSS module name mangling (e.g. because you need to target the global SUI classes) so if a selector contains only classes inside `:global()` it will not benefit from being in a CSS module anymore

Reply:
okay, got it. Thanks for explaining it